components/admin_console/message_export_settings.jsx

```diff
@@ -48,7 +48,7 @@ export default class MessageExportSettings extends AdminSettings {
     renderSettings() {
         const exportFormatOptions = [
             {value: 'actiance', text: Utils.localizeMessage('admin.complianceExport.exportFormat.actiance', 'Actiance XML')},
-            {value: 'globalrelay', text: Utils.localizeMessage('admin.complianceExport.exportFormat.globalrelay', 'GlobalRelay EML')},
+            // {value: 'globalrelay', text: Utils.localizeMessage('admin.complianceExport.exportFormat.globalrelay', 'GlobalRelay EML')},
```

Comment:
I'm using comments, because is only temporary. We can remove completely the code and revert the commit after v4.8 branch cut.